android/run_tests.sh

```diff
@@ -4,42 +4,10 @@ set -eux
 PYTORCH_DIR="$(cd $(dirname $0)/..; pwd -P)"
 PYTORCH_ANDROID_DIR=$PYTORCH_DIR/android
 
-echo "ANDROID_HOME:$ANDROID_HOME"
-if [ ! -z "$ANDROID_HOME" ]; then
```

Comment:
This should be `if [ -z` instead of `if [ ! -z`. It's fixed in the master build script but not here - consolidating these functions can avoid this type of inconsistency.